npm/examples/windows/setkubeconfigpath.ps1

```diff
@@ -0,0 +1,9 @@
+$cpEndpoint = Get-Content C:\k\config | ForEach-Object -Process {if($_.Contains("server:")) {$_.Trim().Split()[1]}}
```

Comment:
Can you add some comments on what/how cpendpoint is or looks like, similarly for other fields. Have a readme.md for this logic, if possible